src/diagnostic/detail/h5writer.h

```diff
@@ -269,7 +269,7 @@ namespace
         if (!file.exist(group_name))
             file.createGroup(group_name);
     }
-}
+} // namespace
```

Comment:
TORM

Reply:
it's a clang format auto thing... can't erase, comes back